_posts/2023-03-20-decommissioning-postgresql-database.adoc

```diff
@@ -0,0 +1,161 @@
+ public | devfile_entrypoint                              | table | pgche
+ public | devfile_entrypoint_arg                          | table | pgche
+ public | devfile_entrypoint_commands                     | table | pgche
+ public | devfile_entrypoint_selector                     | table | pgche
+ public | devfile_env                                     | table | pgche
+ public | devfile_project                                 | table | pgche
+ public | devfile_volume                                  | table | pgche
+ public | environment                                     | table | pgche
+ public | externalmachine                                 | table | pgche
+ public | externalmachine_attributes                      | table | pgche
+ public | externalmachine_env                             | table | pgche
+ public | k8s_runtime_command                             | table | pgche
+ public | k8s_runtime_command_attributes                  | table | pgche
+ public | machine_volume                                  | table | pgche
+ public | preference                                      | table | pgche
+ public | preference_preferences                          | table | pgche
+ public | profile                                         | table | pgche
+ public | profile_attributes                              | table | pgche
+ public | projectattribute                                | table | pgche
+ public | projectattribute_values                         | table | pgche
+ public | projectconfig                                   | table | pgche
+ public | projectconfig_mixins                            | table | pgche
+ public | schema_version                                  | table | pgche
+ public | sequence                                        | table | pgche
+ public | serverconf                                      | table | pgche
+ public | serverconf_attributes                           | table | pgche
+ public | serverconf_properties                           | table | pgche
+ public | sourcestorage                                   | table | pgche
+ public | sourcestorage_parameters                        | table | pgche
+ public | sshkeypair                                      | table | pgche
+ public | user_aliases                                    | table | pgche
+ public | userdevfile                                     | table | pgche
+ public | usr                                             | table | pgche
+ public | workspace                                       | table | pgche
+ public | workspace_attributes                            | table | pgche
+ public | workspaceconfig                                 | table | pgche
+(84 rows)
+----
+
+However, with the switch to link:https://che.eclipseprojects.io/2021/10/12/@mario.loriedo-devfile-v2-and-the-devworkspace-operator-p1.html[DevWorkspace Operator and Devfile v2], all this data become irrelevant since we start using native Kubernetes resources instead.
```

Comment:
```suggestion
However, with the switch to link:https://che.eclipseprojects.io/2021/10/12/@mario.loriedo-devfile-v2-and-the-devworkspace-operator-p1.html[DevWorkspace Operator and Devfile v2], all this data becomes irrelevant since we started using native Kubernetes resources instead.
```